src/components/campaignLandingPage/formatted-type.ts

```diff
@@ -48,4 +49,14 @@ export interface CampaignLandingPage extends PublishedEntity {
     cta: Button | null
     teasers: LinkTeaser[]
   }
+  stories: {
+    show: boolean
+    header: string | null
+    intro: string | null
+    cta: {
+      url: string
+      label: string
+    } | null
```

Comment:
This part was a little funky. I expected that the CTA would be an entity like CTA in other places, in stories it was just an object with basic info.